src/main/scala/devices/spi/TLSPI.scala

```diff
@@ -42,26 +43,49 @@ case class SPIParams(
     frameBits: Int = 8,
     delayBits: Int = 8,
     divisorBits: Int = 12,
-    sampleDelay: Int = 2)
+    fineDelaySelectWidth: Int = 5,
```

Comment:
The initial value has no bearing on what the value will be once software writes it. Software should modify the value to whatever is needed per the synchronization that is present elsewhere.

Reply:
That is not true, because often we boot from the SPI device. So it MUST "Just Work" out of reset given the initial value, clock frequencies, and other synchronization in the design, before SW writes it.

Reply:
Oh, I see.

Reply:
I will make the default value a parameter.